FreeRTOS/FreeRTOS-Kernel
# Add code to allow building for x64 in MSVC

Description
-----------
- Add code for x64 arch.
- Add initial value for local otherwise it won't get proper value in x64.

Test Steps
-----------
- Run MSVC solution in both x86 and x64 build.

Checklist:
----------



Related Issue
-----------
- Required for running on MSVC x64. Demo updated @ FreeRTOS/FreeRTOS#1139.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
